app/services/letter_sending_service.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require "notifications/client"
+
+class LetterSendingService
+  DEFAULT_NOTIFY_TEMPLATE_ID = "701e32b3-2c8c-4c16-9a1b-c883ef6aedee"
+
+  attr_reader :neighbour, :message
+
+  def initialize(neighbour, message)
+    @local_authority = neighbour.consultation.planning_application.local_authority
+    @neighbour = neighbour
+    @message = message
+  end
+
+  def deliver!
+    letter_record = NeighbourLetter.new(neighbour:, text: message)
+    letter_record.save!
+
+    begin
+      response = client.send_letter(
+        template_id: notify_template_id,
+        personalisation: {
+          address: neighbour.address,
```

Comment:
Just reading the docs for my story and it seems you need to split the address into lines https://docs.notifications.service.gov.uk/rest-api.html#send-a-letter, at least 3 lines needed

Reply:
Don't think it'll ever be non-UK so shouldn't have to worry about that? 

Do you mean adding postcode, name to the neighbour model and then passing those as further address lines in this method?

Reply:
Currently the address is a comma-separated string (or maybe that's just what factory_bot gives us) but I think that's ambiguous, since some people put a comma after a house number. I suppose it's not the end of the world if the house number ends up on a separate line but I think it'd be better to treat it as a newline-separated string?

Reply:
Might be good to have a look at what the API we're using gives us - you can test it on staging. Generally seems to be a comma after the house number, but yeah don't think it's end of the world